src/Axis.js

```diff
@@ -998,6 +999,16 @@ export default class Axis extends BaseClass {
     return arguments.length ? (this._tickSize = _, this) : this._tickSize;
   }
 
+  /**
+      @memberof Timeline
+      @desc Allows to set ticks using custom specifiers.  If *value* is not specified, the default time format is returned.
```

Comment:
`@desc Sets the tick specifier for the [tickFormat](https://github.com/d3/d3-scale#continuous_tickFormat) function. If this method is called without any arguments, the default tick specifier is returned.`